test/providers/cloudflare_test.rb

```diff
@@ -111,15 +111,16 @@ def test_build_ptr_from_api
     api_record = {
       "id" => "123462",
       "type" => "PTR",
-      "name" => "4.3.2.1.in-addr.arpa",
-      "content" => "host.record-store-dns-tests.shopitest.com",
+      "name" => "0.183.136.115.in-addr.arpa",
+      "content" => "tbnbz2ni.byoip.example.com",
       "ttl" => 3600
     }
 
     record = @cloudflare.send(:build_from_api, api_record)
 
     assert_kind_of(Record::PTR, record)
-    assert_equal('host.record-store-dns-tests.shopitest.com.', record.ptrdname)
+    assert_equal('tbnbz2ni.byoip.example.com.', record.ptrdname)
```

Comment:
I was expecting to see one test with capital letters like TBnbz2ni... no?

Reply:
that would be this [test](https://github.com/Shopify/record_store/blob/c6fad2005b59ccaa0518a771c5197c4cfb124af6/test/providers/cloudflare_test.rb#L241)